exercises/hamming/hamming_test.exs

```diff
@@ -8,34 +8,46 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule DNATest do
   use ExUnit.Case
 
-  # @tag :pending
+#   @tag :pending
+  test "hamming distance for strands of same length returns :ok" do
+    {code, _} = DNA.hamming_distance('', '')
+    assert code == :ok
+  end
+
+  @tag :pending
+  test "hamming distance is undefined for strands of different lengths" do
+    assert DNA.hamming_distance('AAAC', 'TAGGGGAGGCTAGCGGTAGGAC') == {:error, "Lists must be the same length."}
+    assert DNA.hamming_distance('GACTACGGACAGGACACC', 'GACATCGC') == {:error, "Lists must be the same length."}
+  end
+
+  @tag :pending
   test "no difference between empty strands" do
-    assert DNA.hamming_distance('', '') == 0
+    {:ok, distance} = DNA.hamming_distance('', '')
+    assert distance == 0
```

Comment:
You can also assert on a match. `assert {:ok, 0} = DNA.hamming_distance('','')`. The test failure message will be slightly different.
Or, maybe extract to a custom assertion:

    def assert_distance(one, two, distance) do
      assert {:ok, ^distance} = DNA.hamming_distance(one, two)
    end

    test "no difference between empty strands" do
      assert_distance('', '', 0)
    end